### ISSUE TITLE
Improving code base [DRAFT]

### ISSUE THREAD

**Issue by gliblozhkovyi** [data-skeptic/dataskeptic.com#498]
**TL;DR:**
Small steps which increase site performance and stability. 
Each step could be implemented separated from each other.

### Split backend and frontend servers
Currently backend and frontend servers now are combined. What does that mean?
First of all any backend changes requires full server reload.
In second part webpack (frontend server) blocks backend server request until build the react page sources.

**Advantages.**
Faster site speed. 
Backend errors will not break `master` server.

### Move to wepback 3
We use "old" version (1.x).  Right now build takes over 50 seconds. Even if it was small change like typo fix. 
```
webpack built e1071aecbb2cfd5550a6 in 48462ms
```
**Advantages.**
Faster client-side experience. 
Lower CPU utilization.

### Autoformat code  // delivered at https://github.com/data-skeptic/dataskeptic.com/pull/537
We could use https://github.com/prettier/prettier. After each commit/push it will format the code base to look relative to our standarts. No more `tabs` vs `spaces` or forgotten ";" issues. 

**Advantages.**
Cleaner PR diff look. 
Code in repository have a single convention.
No matter IDE, editor, e.t.c. where you work on.

**Comment by kylepolich** [comment]
@gleblozhkovoy Great suggestions.  Let's work on "Move to webpack 3" and "Autoformat code".  I want to understand more about "Split backend and frontend servers".  Let's talk about it on Tuesday.  I think its probably a good idea, but I want to understand more.

Thank you very much for the pro-active suggestions.
